library/src/main/java/com/google/maps/android/data/kml/KmlStyle.java

```diff
@@ -187,6 +187,8 @@ public String getIconUrl() {
      * @param color Fill color for a KML Polygon as a String
      */
     /* package */ void setFillColor(String color) {
+        // Tolerate KML with leading or trailing whitespace in colors
+        color = color.trim();
```

Comment:
How about applying this method inside `convertColor` to avoid duplication and to make sure the same input sanitation is applied? Looks like in all cases `color.trim()` is desired before calling `convertColor` anyway.

Reply:
Makes sense! I just refactored this accordingly in https://github.com/googlemaps/android-maps-utils/pull/599/commits/5074520c342ef46fc6eb82f002918609a4f0c350.